src/main/java/com/github/shyiko/mysql/binlog/event/deserialization/FormatDescriptionEventDataDeserializer.java

```diff
@@ -32,7 +33,9 @@ public FormatDescriptionEventData deserialize(ByteArrayInputStream inputStream)
         eventData.setServerVersion(inputStream.readString(50).trim());
         inputStream.skip(4); // redundant, present in a header
         eventData.setHeaderLength(inputStream.readInteger(1));
-        // lengths for all event types
+        // skip lengths for some event types
+        inputStream.skip(EventType.FORMAT_DESCRIPTION.ordinal() - 1);
```

Comment:
can you please explain this? why EventType.FORMAT_DESCRIPTION.ordinal()?

Reply:
done